modules.go

```diff
@@ -7,5 +7,5 @@ import (
 	_ "github.com/gliderlabs/logspout/routesapi"
 	_ "github.com/gliderlabs/logspout/transports/tcp"
 	_ "github.com/gliderlabs/logspout/transports/udp"
-	_ "github.com/iamatypeofwalrus/logspout-loggly/loggly"
+	_ "github.com/uaudio/logspout-loggly/loggly"
```

Comment:
Should be merged in and built by 9pm tonight. 

> On Apr 14, 2016, at 6:24 PM, David Multer <notifications@github.com> wrote:
> 
> In modules.go:
> 
> > @@ -7,5 +7,5 @@ import (
> >  	_ "github.com/gliderlabs/logspout/routesapi"
> >  	_ "github.com/gliderlabs/logspout/transports/tcp"
> >  	_ "github.com/gliderlabs/logspout/transports/udp"
> > -	_ "github.com/iamatypeofwalrus/logspout-loggly/loggly"
> > +	_ "github.com/uaudio/logspout-loggly/loggly"
> Thanks for moving so fast on this. Any idea when you think it might merge and pushed to Docker Hub?
> 
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
>